common/Cargo.toml

```diff
@@ -27,13 +27,16 @@ regex = { workspace = true }
 reqwest = { workspace = true, features = ["native-tls"] }
 ring = { workspace = true }
 schemars = { workspace = true }
-sea-orm = { workspace = true, features = ["sea-query-binder", "sqlx-postgres", "runtime-tokio-rustls", "macros"] }
+sea-orm = { workspace = true, features = [
```

Comment:
I'd like to keep the one-liners, as this makes sorting the deps alphabetically easier.